tests/testconfig/.gitlab-ci.yml

```diff
@@ -72,13 +72,15 @@ before_script:
   # build
   - time cmake --build . --target ${BUILD_TARGETS} -- -j `nproc` 2>&1 | tee ${CI_PROJECT_DIR}/build.log
 
-.build-and-test: &build-and-test
-  - *build-only
-  # test
+.test-only: &test-only
```

Comment:
Please drop this change and perform the respective changes in the Github CI files.